src/apps/routes/tools.clj

```diff
@@ -16,11 +17,13 @@
                            user-get-tool]]
         [apps.tools.private :only [add-private-tool delete-private-tool update-private-tool]]
         [apps.user :only [current-user]]
+        [apps.util.coercions :only [coerce!]]
         [apps.util.service]
         [slingshot.slingshot :only [throw+]]
         [ring.util.http-response :only [ok]])
   (:require [apps.routes.schemas.permission :as permission]
             [apps.service.apps :as apps]
+            [apps.service.apps.de.listings :as app-listings]
```

Comment:
Yeah, Agave doesn't have the concept of a tool, which makes this unnecessary for now. I'm not sure whether or not we'll ever integrate a third-party execution platform that supports separate tools and apps, so it may never be necessary from a functional standpoint.

The other reason I'd like to have all requests go through that interface is kind of aesthetic. Limiting calls to that namespace hierarchy makes it slightly easier to refactor the code in the future if we need to.

This isn't a huge deal, and certainly isn't something that we need to tackle now; it's just something that I'd like to do sometime.